src/components/TileComponents.js

```diff
@@ -0,0 +1,29 @@
+import { render } from '../shared/dom';
+
+export const TileComponent = ({ title, image }) => {
+    return `
+    <div class="movie-tile">
+        <img src="${image}"/>
+        <p class='title'>${title}</p>
```

Comment:
title is actually not a paragraph, it should be Hx tag (h3 for example)

Reply:
paragraph tag has been changed for H3 tag, I didn't edit the style for this tag, we can do it later in SCSS file